api/hiveextension/v1beta1/agentclusterinstall_types.go

```diff
@@ -210,6 +210,13 @@ type Networking struct {
 	// +kubebuilder:validation:MaxItems=1
 	// +optional
 	ServiceNetwork []string `json:"serviceNetwork,omitempty"`
+
+	//NetworkType is the Container Network Interface (CNI) plug-in to install
+	//The default value is OpenShiftSDN for IPv4 and OVNKubernetes for IPv6
```

Comment:
How and when is the default determined and applied?

Reply:
I reinstated the function that determine the default before reconciliation